scripts/Diffraction/isis_powder/osiris.py

```diff
@@ -153,9 +177,80 @@ def _get_drange_for_run_number(self, run_number_string):
                 return drange_name
         return None
 
+    def _apply_absorb_corrections(self, run_details, ws_to_correct):
+        """
+        Generates absorption corrections using monte carlo absorption.
+        :param ws_to_correct: workspace that needs to be corrected.
+        :param run_details: the run details of the workspace. Unused parameter added for API compatibility.
+        :return: A workspace containing the corrections.
+        """
+        events_per_point = 1000
```

Comment:
should this be hard coded or an argument with a default?

Reply:
It could be a parameter. I will add it as a parameter